cli/src/main/java/org/apache/uniffle/cli/UniffleCLI.java

```diff
@@ -66,14 +91,40 @@ public int run(String[] args) throws UniffleCliArgsException {
       System.out.println("uniffle-admin-cli : " + cliArgs);
       return 0;
     }
+    if (cmd.hasOption(coordServer.getOpt()) && cmd.hasOption(coordPort.getOpt())) {
+      String host = cmd.getOptionValue(coordServer.getOpt()).trim();
+      int port = Integer.parseInt(cmd.getOptionValue(coordPort.getOpt()).trim());
+      String hostUrl = String.format("http://%s:%d", host, port);
```

Comment:
It is also possible to use https.

Reply:
> It is also possible to use https.

I don't quite understand, could you provide more context or hints?

Reply:
https://github.com/apache/incubator-uniffle/blob/b8f17bf588aa2333aa2f4c4d97d5f6b149a63bcd/common/src/main/java/org/apache/uniffle/common/web/JettyServer.java#L77-L79

Reply:
> https://github.com/apache/incubator-uniffle/blob/b8f17bf588aa2333aa2f4c4d97d5f6b149a63bcd/common/src/main/java/org/apache/uniffle/common/web/JettyServer.java#L77-L79

Thank you, I will make some changes accordingly.